MaterialLib/Component/Diffusion/createDiffusion.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file:   createDiffusion.h
+ */
+
+#pragma once
+
+#include <memory>
+#include "ProcessLib/Parameter/Parameter.h"
```

Comment:
Forward declaration of ParameterBase as:
```
namespace ProcessLib
{
struct ParameterBase;
}
```

Reply:
Done.